cmd/drone-gcr/main.go

```diff
@@ -61,12 +60,52 @@ func main() {
 	cmd := exec.Command(docker.GetDroneDockerExecCmd())
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
-	err = cmd.Run()
+	err := cmd.Run()
 	if err != nil {
 		logrus.Fatal(err)
 	}
 }
 
+func getOauthToken(data []byte) (s string) {
+	scopes := []string{
+		"https://www.googleapis.com/auth/cloud-platform",
+	}
+	ctx := context.Background()
+	credentials, err := google.CredentialsFromJSON(ctx, data, scopes...)
+	if err == nil {
+		token, err := credentials.TokenSource.Token()
+		if err == nil {
+			return token.AccessToken
+		}
+	}
+	return
+}
+
+func setUsernameAndPassword(user string, pass string, is_wi bool) (u string, p string) {
```

Comment:
golang uses camelcase or pascal case for variable names depending on context, it should not use snake case. 
https://gyulgrigoryan.medium.com/variables-in-golang-1bf492df03b0
so `is_wi` can you rename it to `workloadIdentitiy"